src/main/java/com/exactpro/th2/check1/event/VerificationEntryUtils.java

```diff
@@ -92,14 +91,14 @@ private static VerificationStatus toVerificationStatus(StatusType statusType) {
         }
 
         switch (statusType) {
-        case NA:
-            return VerificationStatus.NA;
-        case FAILED:
-            return VerificationStatus.FAILED;
-        case PASSED:
-            return VerificationStatus.PASSED;
-        default:
-            throw new IllegalArgumentException("Unsupported status type '" + statusType + '\'');
+            case NA:
+                return VerificationStatus.NA;
+            case FAILED:
+                return VerificationStatus.FAILED;
+            case PASSED:
+                return VerificationStatus.PASSED;
+            default:
+                throw new IllegalArgumentException("Unsupported status type '" + statusType + '\'');
```

Comment:
I believe we must stop changing formatting in the places where we don't even change the code...

Reply:
Thank you for your comment, I have changed IDE configuration